src/ExtensionsSample/Samples/DocumentDBSamples.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using ExtensionsSample.Models;
+using Microsoft.Azure.WebJobs;
+
+namespace ExtensionsSample.Samples
+{
+    // To use the DocumentDB samples:
+    // 1. Create a new DocumentDB account
+    // 2. Add the DocumentDB Endpoint Uri to the 'AzureWebJobsDocumentDBUri' App Setting in app.config
+    // 3. Add the DocumentDB Primary Key to the 'AzureWebJobsDocumentDBKey' App Setting in app.config
+    // 4. Add typeof(DocumentDBSamples) to the SamplesTypeLocator in Program.cs
+    public static class DocumentDBSamples
+    {
+        // POCO Output binding
+        //   The binding inserts the newly created item into the specified Collection when the
+        //   method successfully exits.
+        //   If the database or the collection do not exist, they are automatically created.
```

Comment:
What is the default for CreateIfNotExists? Many/most of our bindings default to creating the output queue/table/container if it doesn't exist (see http://go.microsoft.com/fwlink/?LinkID=524028&clcid=0x409). I think yours should default to true, then you don't have to specify explicitly below.